### ISSUE TITLE
[Paper Note] Xiaomi MiMo-VL-Miloco Technical Report, Jiaze Li+, arXiv'25, 2025.12

### ISSUE THREAD

**Issue by AkihikoWatanabe** [AkihikoWatanabe/paper_notes#4048]
# URL
- https://arxiv.org/abs/2512.17436
# Authors
  - Jiaze Li
  - Jingyang Chen
  - Yuxun Qu
  - Shijie Xu
  - Zhenru Lin
  - Junyou Zhu
  - Boshen Xu
  - Wenhui Tan
  - Pei Fu
  - Jianzhong Ju
  - Zhenbo Luo
  - Jian Luan
# Abstract
  - We open-source MiMo-VL-Miloco-7B and its quantized variant MiMo-VL-Miloco-7B-GGUF, a pair of home-centric vision-language models that achieve strong performance on both home-scenario understanding and general multimodal reasoning. Built on the MiMo-VL-7B backbone, MiMo-VL-Miloco-7B is specialized for smart-home environments, attaining leading F1 scores on gesture recognition and common home-scenario understanding, while also delivering consistent gains across video benchmarks such as Video-MME, Video-MMMU, and Charades-STA, as well as language understanding benchmarks including MMMU-Pro and MMLU-Pro. In our experiments, MiMo-VL-Miloco-7B outperforms strong closed-source and open-source baselines on home-scenario understanding and several multimodal reasoning benchmarks. To balance specialization and generality, we design a two-stage training pipeline that combines supervised fine-tuning with reinforcement learning based on Group Relative Policy Optimization, leveraging efficient multi-domain data. We further incorporate chain-of-thought supervision and token-budget-aware reasoning, enabling the model to learn knowledge in a data-efficient manner while also performing reasoning efficiently. Our analysis shows that targeted home-scenario training not only enhances activity and gesture understanding, but also improves text-only reasoning with only modest trade-offs on document-centric tasks. Model checkpoints, quantized GGUF weights, and our home-scenario evaluation toolkit are publicly available at https://github.com/XiaoMi/xiaomi-mimo-vl-miloco to support research and deployment in real-world smart-home applications.
# Translation (by gpt-4o-mini)
- 私たちは、MiMo-VL-Miloco-7Bおよびその量子化バリアントであるMiMo-VL-Miloco-7B-GGUFをオープンソース化しました。これらは、家庭中心の視覚と言語のモデルのペアであり、家庭シナリオの理解と一般的なマルチモーダル推論の両方で優れた性能を発揮します。MiMo-VL-7Bバックボーンに基づいて構築されたMiMo-VL-Miloco-7Bは、スマートホーム環境に特化しており、ジェスチャー認識や一般的な家庭シナリオの理解において先進的なF1スコアを達成しています。また、Video-MME、Video-MMMU、Charades-STAなどのビデオベンチマークや、MMM-Pro、MMLU-Proなどの言語理解ベンチマークでも一貫した向上を示しています。私たちの実験では、MiMo-VL-Miloco-7Bが家庭シナリオの理解やいくつかのマルチモーダル推論ベンチマークにおいて、強力なクローズドソースおよびオープンソースのベースラインを上回ることが確認されました。特化と一般性のバランスを取るために、効率的なマルチドメインデータを活用したグループ相対ポリシー最適化に基づく強化学習と監視付きファインチューニングを組み合わせた二段階のトレーニングパイプラインを設計しました。さらに、チェーン・オブ・ソートの監視とトークン予算を考慮した推論を組み込むことで、モデルがデータ効率的に知識を学びながら、効率的に推論を行えるようにしています。私たちの分析では、ターゲットを絞った家庭シナリオのトレーニングが、活動やジェスチャーの理解を向上させるだけでなく、文書中心のタスクにおいてもわずかなトレードオフでテキストのみの推論を改善することが示されています。モデルのチェックポイント、量子化されたGGUFウェイト、および家庭シナリオ評価ツールキットは、実世界のスマートホームアプリケーションにおける研究と展開を支援するために、https://github.com/XiaoMi/xiaomi-mimo-vl-miloco で公開されています。
# Summary (by gpt-4o-mini)
- MiMo-VL-Miloco-7Bとその量子化バリアントをオープンソース化し、家庭中心の視覚と言語モデルとして優れた性能を発揮。特にスマートホーム環境に特化し、ジェスチャー認識やマルチモーダル推論で高いF1スコアを達成。二段階のトレーニングパイプラインを設計し、効率的な推論を実現。家庭シナリオのトレーニングが活動理解を向上させ、テキスト推論にも効果を示す。モデルとツールキットは公開され、スマートホームアプリケーションの研究に貢献。

**Comment by AkihikoWatanabe** [comment]
元ポスト:https://x.com/gm8xx8/status/2003092118277853553?s=46&t=Y6UuIHB0Lv0IpmFAjlc2-Q

**Comment by AkihikoWatanabe** [comment]
HF:https://huggingface.co/collections/xiaomi-open-source/xiaomi-mimo-vl-miloco

モデル自体は11月から公開されている

**Comment by AkihikoWatanabe** [comment]
home-scenario gesture recognitionとdaily activity recognitionでGemini-2.5-Proを上回る性能を達成している。特定のユースケースに特化しつつ、genericなユースケースの性能を損なわないようなモデルを学習したい場合は参考になるかもしれない。

<img width="852" height="470" alt="Image" src="https://github.com/user-attachments/assets/91055d6f-0247-469c-96e9-c1372133c9d8" />

まずSFTでhome-scenarioデータ[^1] + GeneralデータのDataMixでreasoning patternを学習させ、tokenのefficiencyを高めるためにCoTパターンを排除しdirect answerをするようなデータ（およびprompting）でも学習させる。これによりhome-scenarioでの推論能力が強化される。SFTはfull parameter tuningで実施され、optimizerはAdamW。バッチサイズ128, warmup ratio 0.03, learning rate 1 * 10^-5。スケジューラについては記述がないように見える。

その後、一般的なユースケース（Video Understanding (temporal groundingにフォーカス), GUI Grounding, Multimodal Reasoning （特にSTEMデータ））データを用いてGRPOでRLをする。明らかに簡単・難しすぎるデータは除外。RLのrewardは `r_acc + r_format`の線形補完（係数はaccL: 0.9, format: 0.1）で定義される。r_accはデータごとに異なっている。Video Understandingでは予測したqueryに対してモデルが予測したtimespanとgoldのtimespanのoverlapがどの程度あるかをaccとし、GUI Groundingではbounding boxを予測しpred/goldのoverlapをaccとする。Multimodal ReasoninghはSTEMデータなので回答が一致するかをbinaryのaccとして与えている。

モデルのアーキテクチャは、アダプターでLLMと接続するタイプのもので、動画/画像のBackboneにはViTを用いて、MLPのアダプターを持ちいてLLMの入力としている。
<img width="861" height="500" alt="Image" src="https://github.com/user-attachments/assets/6550be7f-6c48-4189-9f38-02eabe2e17b4" />


[^1]: volunteerによるhome-scenarioでのデータ作成; ruleを規定しvolunteerに理解してもらいデータ収集。その後研究者が低品質なものを除外
